cmd/bucket-stats.go

```diff
@@ -27,6 +31,85 @@ type BucketStats struct {
 // BucketReplicationStats represents inline replication statistics
 // such as pending, failed and completed bytes in total for a bucket
 type BucketReplicationStats struct {
+	Stats map[string]*BucketReplicationStat
```

Comment:
this is used for the in-memory replication metrics cache and metrics reporting API. Made it a pointer because individual in-memory target stats need to be accumulated with cluster stats and this makes it easier to write a helper function for that.

Reply:
👍🏼